driver-core/src/main/com/mongodb/client/model/expressions/BooleanExpression.java

```diff
@@ -19,49 +19,47 @@
 import java.util.function.Function;
 
 /**
- * Expresses a boolean value.
+ * A logical boolean value, either true or false.
```

Comment:
The redundancy is there because I seem to be taking the opportunity to remind users that "null" is not a valid value. But I have removed this.

I thought that capitalizing would indicate that it is a Java Boolean, while wrapping in `code` would indicate it is the primitive, but leaving it uncapitalized would be in line with speaking abstractly, which is what I think we are trying to do here? And if we are, I don't think it is usually capitalized, or perhaps that is that not the case?

Reply:
Abstract Boolean type and logic are capitalized similarly to Newtonian mechanics.

Reply:
Resolving to avoid wasting time discussing things that are not obvious bugs.